VAMobile/documentation/docs/Flagship design library/Templates/Home.md

```diff
@@ -25,7 +25,7 @@ Having a mix of content, arranged into intentional zones, ensures that users:
 
 The VA Mobile app Home screen zones ensure that the most impactful content items are getting the proper placement in the screen's hierarchy while also handling a variety of user contexts and technical situations. 
 
-<img src="/workspaces/va-mobile-app/VAMobile/documentation/static/img/home/vamobile-home-zones.png" width="700" alt="Diagram of home screen zones for VA mobile app" />
+<img src="https://department-of-veterans-affairs.github.io/va-mobile-app/img/home/vamobile-home-zones.png" width="700" alt="Diagram of home screen zones for VA mobile app" />
```

Comment:
Think there is a way to reference this without the full URL since it is a local asset, but haven't mucked with images much in the doc site--the URL shouldn't be changing so it should be fine though if it works.